NuGet/Realm/Realm.nuspec

```diff
@@ -17,10 +17,10 @@
     <tags>Realm database db persistence sqlite</tags>
     <dependencies>
       <group targetFramework="portable-net45+sl5+wp8+wpa81+win8+monoandroid+Xamarin.iOS10+monotouch+Xamarin.Mac">
-        <dependency id="Realm.Database" version="$version$"/>
+        <dependency id="Realm.Database" version="$version$" exclude="native" />
```

Comment:
Should we start putting the iOS and Android artifacts in the runtimes folder? Seems like that would allow us to get rid of the workaround in the targets file.